_includes/nav.html

```diff
@@ -13,29 +13,21 @@
       <ul class="nav navbar-nav">
         <li class="nav-link"><a href="{{ site.baseurl }}/downloads">Download</a></li>
         <li class="dropdown">
+        <a class="dropdown-toggle" data-toggle="dropdown" href="#">Releases<span class="caret"></span></a>
+        <ul class="dropdown-menu">
+          <li><a href="{{ site.baseurl }}/release/1.8.0/">1.8.0 (Latest)</a></li>
+          <li><a href="{{ site.baseurl }}/release/1.7.2/">1.7.2</a></li>
+          <li><a href="{{ site.baseurl }}/release/1.6.6/">1.6.6</a></li>
+          <li><a href="{{ site.baseurl }}/release/">Archive</a></li>
+        </ul>
+        </li>
+        <li class="dropdown">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#">Documentation<span class="caret"></span></a>
         <ul class="dropdown-menu">
-          <li class="dropdown-header">1.8.0 Release (Latest)</li>
-          <li><a href="{{ site.baseurl }}/release_notes/1.8.0">Release Notes</a></li>
-          <li><a href="{{ site.baseurl }}/1.8/accumulo_user_manual">User Manual</a></li>
-          <li><a href="{{ site.baseurl }}/1.8/apidocs">Javadoc</a></li>
-          <li><a href="{{ site.baseurl }}/1.8/examples">Examples</a></li>
-          <li class="divider"></li>
-          <li class="dropdown-header">1.7.2 Release</li>
-          <li><a href="{{ site.baseurl }}/release_notes/1.7.2">Release Notes</a></li>
-          <li><a href="{{ site.baseurl }}/1.7/accumulo_user_manual">User Manual</a></li>
-          <li><a href="{{ site.baseurl }}/1.7/apidocs">Javadoc</a></li>
-          <li><a href="{{ site.baseurl }}/1.7/examples">Examples</a></li>
-          <li class="divider"></li>
-          <li class="dropdown-header">1.6.6 Release</li>
-          <li><a href="{{ site.baseurl }}/release_notes/1.6.6">Release Notes</a></li>
-          <li><a href="{{ site.baseurl }}/1.6/accumulo_user_manual">User Manual</a></li>
-          <li><a href="{{ site.baseurl }}/1.6/apidocs">Javadoc</a></li>
-          <li><a href="{{ site.baseurl }}/1.6/examples">Examples</a></li>
-          <li class="divider"></li>
+          <li><a href="{{ site.baseurl }}/user-manual/">User Manuals</a></li>
+          <li><a href="{{ site.baseurl }}/javadocs/">Javadocs</a></li>
+          <li><a href="{{ site.baseurl }}/examples/">Accumulo Examples</a></li>
```

Comment:
I'm not a fan of the addition of these three items. I feel that we should be working to minimize the clutter of our navigation menus, and I think individual release pages should have links to their relevant docs. Any docs in the menu should be basic, introductory docs, not full-fledged APIs and book-like user manuals that a new user is going to find overwhelming.

More items in the menu means users spend more time looking through the menus than reading page contents. While this may save some people time later, if they know exactly what they want, I don't think the benefits to those people outweigh the extra time spent for *all* users who have to navigate the menu.

Improvements here could be as a follow-on action.